pkg/kubelet/kubelet_node_status.go

```diff
@@ -597,18 +596,14 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 			}
 		}
 
-		devicePluginCapacity, devicePluginAllocatable, removedDevicePlugins = kl.containerManager.GetDevicePluginResourceCapacity()
+		devicePluginCapacity, devicePluginAllocatable = kl.containerManager.GetDevicePluginResourceCapacity()
```

Comment:
Given that https://github.com/kubernetes/kubernetes/pull/60332 is meant to be a temporary solution till we can get more specific information about cluster-level resource from resource API, I would like to avoid the interface changes along the stack. The original change I was thinking is to not populate the deletedResources field in GetCapacity() function, but I think that change would cause the device plugin e2e_node test failure because we checked whether node capacity is zero or gone after removing the device plugin. Thinking about this, perhaps the simplest change is to  set capacity to zero for removedResource below. That will allow the e2e_node test to continue passing, gives us a clear signal to differentiate device plugin failure vs device failure, and if we need to, can be easily changed back.

Reply:
FYI, looks like the suggested change has been included in https://github.com/kubernetes/kubernetes/pull/60332